openapi.yaml

```diff
@@ -208,10 +497,378 @@ definitions:
+    enum:
+      - game
+  MaximumBet:
+    type: integer
+    minimum: 1
+    description: The maximum bet every player can place each round.
+    example: 2000
+  MinimumBet:
+    type: integer
+    minimum: 1
+    description: The minimum bet every player have to place at least each round.
+    example: 1000
+  PlayerBet:
+    type: integer
+    minimum: 0
+    description: The bet from a specific player.
+  MemberEmail:
+    type: string
+    example: abc@efg.com
+    format: email
+  MemberExpAboveLevel:
+    type: integer
+    minimum: 0
+    description: Player's experience point above current level.
+    example: 1234
+  MemberId:
+    type: string
+    example: '101'
+  MemberIds:
+    type: array
+    items:
+      $ref: '#/definitions/MemberId'
+  PlayerId:
+    type: string
+    example: 'g1m1'
+  PlayerIds:
+    type: array
+    items:
+      $ref: '#/definitions/PlayerId'
+  PlayerInGamePatchBody:
+    properties:
+      data:
+        type: object
+        properties:
+          id:
+            $ref: '#/definitions/PlayerId'
+          type:
+            $ref: '#/definitions/PlayerType'
+          attributes:
+            type: object
+            properties:
+              playerBet:
+                $ref: '#/definitions/PlayerBet'
+              playerStatus:
+                $ref: '#/definitions/PlayerStatus'
+              playerCards:
+                $ref: '#/definitions/Cards'
+            additionalProperties: false
+        required:
+          - id
+          - type
+        additionalProperties: false
+    required:
+      - data
+    additionalProperties: false
+  PlayerInGameResource:
+    properties:
+      id:
+        $ref: '#/definitions/PlayerId'
+      type:
+        $ref: '#/definitions/PlayerType'
+      attributes:
+        type: object
+        properties:
+          memberId:
+            $ref: '#/definitions/MemberId'
+          memberName:
+            $ref: '#/definitions/MemberName'
+          memberLevel:
+            $ref: '#/definitions/MemberLevel'
+          memberExpAboveLevel:
+            $ref: '#/definitions/MemberExpAboveLevel'
+          playerBet:
+            $ref: '#/definitions/PlayerBet'
+          playerStatus:
+            $ref: '#/definitions/PlayerStatus'
+          playerCards:
+            $ref: '#/definitions/Cards'
+  PlayerInGameResult:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        $ref: '#/definitions/PlayerInGameResource'
+  PlayerInGameResults:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        type: array
+        items:
+          $ref: '#/definitions/PlayerInGameResource'
+  PlayerPostBody:
+    properties:
+      data:
+        type: object
+        properties:
+          type:
+            $ref: '#/definitions/PlayerType'
+          attributes:
+            type: object
+            properties:
+              memberId:
+                $ref: '#/definitions/MemberId'
```

Comment:
Should the post body also allow `playerBet`, `playerStatus`, and `playerCards`? You could make these optional and add default values for them if you want. This could make it easier to create a player with specific starting attributes.